lib/src/base-styles/_nx-card.scss

```diff
@@ -14,91 +14,62 @@
   border-radius: $nx-border-radius;
   box-sizing: border-box;
   margin-bottom: $nx-spacing-md;
-  padding: $nx-spacing-l $nx-spacing-xxl;
-}
-
-.nx-card__call-out {
-  @include bold();
-
-  font-size: 32px;
-
-  .nx-icon--card-call-out {
-    font-size: 48px;
-  }
-
-  .nx-icon--card-call-out-xl {
-    font-size: 64px;
-  }
+  padding: $nx-spacing-l;
+  text-align: center;
 }
 
 .nx-card-container--row {
+  @include container-horizontal;
   display: flex;
-  -ms-flex-wrap: wrap;
   flex-wrap: wrap;
+  justify-content: space-between;
   margin-bottom: 0;
+}
 
-  + .nx-card-container--row {
-    margin-top: 0;
-  }
-
-  .nx-card {
-    @include container-vertical;
-    display: flex;
-    flex-direction: column;
-    -ms-flex-pack: justify;
-    justify-content: space-between;
-    margin-right: $nx-spacing-md;
-    text-align: center;
-    min-width: 230px;
-    max-width: 330px;
-  }
-
-  .nx-card__header {
-    @include container-vertical;
-    margin-bottom: $nx-spacing-md;
-  }
+.nx-card {
+  @include container-vertical;
+  display: grid;
+  grid-template-rows: auto 1fr auto;
+  margin-right: $nx-spacing-md;
+  min-width: 230px;
+  max-width: 330px;
+}
 
-  .nx-card__call-out {
-    + .nx-card__content {
-      margin-top: $nx-spacing-md;
-    }
-  }
+.nx-card__header {
+  @include container-vertical;
+  margin-bottom: $nx-spacing-md;
+}
 
-  .nx-card__call-out--text-only {
-    + .nx-card__content {
-      margin-top: $nx-spacing-xs;
-    }
-  }
+.nx-card__content {
+  align-content: start;
+  align-items: start;
+  display: grid;
+  row-gap: $nx-spacing-md;
+}
 
-  .nx-card__footer {
-    border-top: $nx-border-default;
-    margin-top: $nx-spacing-l;
-    padding-top: $nx-spacing-l;
-    width: 100%;
+.nx-card__footer {
+  border-top: $nx-border-default;
+  margin-top: $nx-spacing-l;
+  padding-top: $nx-spacing-l;
+  width: 100%;
 
-    > .nx-btn-bar {
-      justify-content: center;
-    }
+  > .nx-btn-bar {
+    justify-content: center;
   }
 }
 
-.nx-card-container--column {
+.nx-card--horizontal {
   @include container-vertical;
-
-  .nx-card {
-    display: -ms-grid;
-    display: grid;
-    -ms-grid-columns: auto 1fr auto;
-    grid-template-columns: auto 1fr auto;
-    -ms-grid-rows: auto auto;
-    grid-template-areas:
-      "card-callout card-title card-footer"
-      "card-callout card-text card-footer";
-  }
+  display: grid;
+  grid-template-columns: auto 1fr auto;
+  grid-template-areas:
+    "card-callout card-title card-actions"
+    "card-callout card-content card-actions";
```

Comment:
What's the purpose of prefixing all of these with "card"?

Reply:
> What's the purpose of prefixing all of these with "card"?

No purpose, removed here: https://github.com/sonatype/sonatype-react-shared-components/pull/307/commits/848980e874c7c3a447e1b31ebe5da962a4789218